packages/@guardian/source-package-validation/validate-types.ts

```diff
@@ -0,0 +1,183 @@
+import type {
```

Comment:
could this just be `export`?
```suggestion
export type {
```

Reply:
Good thinking! Done in https://github.com/guardian/source/pull/985/commits/5122ba3e3f135cdc62063c08e77c8e5de1ac630b